dist/preview release/what's new.md

```diff
@@ -4,6 +4,7 @@
 
 - Added support for [parallel shader compilation](https://www.khronos.org/registry/webgl/extensions/KHR_parallel_shader_compile/) ([Deltakosh](https://github.com/deltakosh))
 - Added FlyCamera for free navigation in 3D space, with a limited set of settings ([Phuein](https://github.com/phuein))
+- Added Object Based Motion Blur post-process ([julien-moreau](https://github.com/julien-moreau))

```

Comment:
Please update it with the doc when done ;)

Reply:
Will put the link when done then :)